src/gui/EffectView.cpp

```diff
@@ -45,65 +49,60 @@
 namespace lmms::gui
 {
 
-EffectView::EffectView( Effect * _model, QWidget * _parent ) :
-	PluginView( _model, _parent ),
-	m_bg( embed::getIconPixmap( "effect_plugin" ) ),
-	m_subWindow( nullptr ),
+EffectView::EffectView(Effect * _model, QWidget * _parent) :
+	PluginView(_model, _parent),
+	m_subWindow(nullptr),
 	m_controlView(nullptr),
 	m_dragging(false)
 {
-	setFixedSize(EffectView::DEFAULT_WIDTH, EffectView::DEFAULT_HEIGHT);
 	setFocusPolicy(Qt::StrongFocus);
+	setViewWidth(EffectView::DEFAULT_WIDTH);
```

Comment:
Consider moving this line to above, reduces the diff.

Reply:
Looked again, why did you abstract this into another function, the logic seems to have unchanged.

Reply:
This was a holdover from the previous implementation where I was trying to make it resizable. It has been removed.